### PR TITLE
Change text to explain merging of accounts

### PR DESCRIPTION
**Pull request by kch8qx**
Update language in account settings to match the discussion from PR #4295

### CODE REVIEW

File: website/templates/profile/account.mako [hunk 1]
```diff
@@ -102,7 +104,7 @@
                         </form>
                     </div>
                 </div>
-				<div class="panel panel-default">
+<div class="panel panel-default">
```
Comment (abought) [inline website/templates/profile/account.mako]: Very tiny nitpick, but could you make sure that the nesting of this div is appropriate?

**Comment by abought** [comment]
CR2 complete (CR1 was done on previous PR).

No language or syntax notes. I might prefer to see a bit more spacing between the emails and explanatory text (see screenshot), but don't know that I'd push for the change in this PR. It's a small thing.

screen shot 2015-09-21 at 3 46 32 pm

**Comment by abought** [comment]
Also, unit tests are passing for me when run locally; not sure why the build is failing.

File: website/templates/profile/account.mako [hunk 2]
```diff
@@ -66,12 +65,15 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
+                                        <p>To merge an existing OSF account with this one or to be able to log in under multiple email addresses, add your alternate email address in the field below.</p>
```
Comment (sloria) [inline website/templates/profile/account.mako]: Possible revision:

> To merge an existing account with this one or log in with multiple email addresses, add an alternate email address  below.

Although, perhaps we don't need to mention "merging" at all? As a user, I would assume adding an alternate email that is already associated with OSF account would get merged. Also, I believe the messaging that follows this step makes it clear that a merge is happening.

If that assumption holds, then this text might not even be necessary.

Pinging @erinmayhood for review.

Comment (abought) [inline website/templates/profile/account.mako]: Honestly? As a user, if I try to register an email address already is use, I'd expect an error message. (not a magic link a la [#OSF-4542]) The merge behavior was a little surprising to me at first discovery, and very powerful relative to the amount of fanfare.

There was some discussion of text on a previous round- see the associated JIRA issue.

Comment (sloria) [inline website/templates/profile/account.mako]: @abought Fair enough; I think having the text is fine if it adds clarity, which it does.

Still wouldn't hurt to have @erinmayhood to do a quick review of the text, as the discussion on the ticket happened after the product demo stage.

Comment (erinmayhood) [inline website/templates/profile/account.mako]: @sloria your tweak is fine with me.

Comment (sloria) [inline website/templates/profile/account.mako]: Will change when I merge.

**Pull request merged**